requirements/server-load-permission-requirements.md

```diff
@@ -31,22 +29,22 @@ During the Farm Load process SPDocKit will try to retrieve information about you
   * Best Practices -> Hardware Requirements -> Free Disk Space
   * Best Practices -> Hardware Requirements -> RAM
   * Best Practices -> Servers -> Hotfixes per Server Role -> all reports
-1. To retrieve a list of available Windows updates please make sure that Windows Update service is up and running. If the service is disabled or not running the following reports will not be available:
+2. To retrieve a list of available Windows updates please make sure that Windows Update service is up and running. If the service is disabled or not running the following reports will not be available:
   * Farm Explorer -> Servers in Farm -> Available Windows Updates
   * Best Practices -> Updates -> Servers -> Windows Updates
 
 #### Solution
 
 1. Add your SPDocKit user accounts to a __Local Administrators__ group on the specified server.
 In case that you have a very strict security policy and you cannot add this account to the __Local Administrators__ group you can use this procedure to get most of reports working:
```

Comment:
In case you have (bez that) 
and cannot add this (bez you)
zarez iza group
most of **the** reports working: